index.html

```diff
@@ -27,8 +27,16 @@
               <li class="nav-link">Contact</li>
             </ul>
         </nav>
+        <nav class="portfolioM">
+          <p class="portfolioB"><i class="fa fa-times"></i></p>
+          <ul class="portfoliomenu">
+            <li class="menu"><a href="#hero" id="mbmu">portfolio</a></li>
+            <li class="menu"><a href="#myself" id="mbmu">About</a></li>
+            <li class="menu"><a href="#form" id="mbmu">contact</a></li>
+          </ul>
+          </nav>  
     </header>
-    <section class="homesection">
+    <section class="homesection" id="hero">
       <div class="desktop-home">
 
         <div>
```

Comment:
- [ ] You have done great so far. :+1: :100:  But as pointed out by the previous reviewer, your mobile menu is not yet fully responsive, looking at the screenshot below, you can see you have a space at the bottom of the menu that is not in the Figma design, the Mobile menu is meant to cover the full screen, I'll suggest you fix this. :+1: :100: 

- - SCREENSHOT:
 
![Selection_066](https://github.com/Danny-imo/my-project/assets/107632780/37210823-af27-457e-a54e-9cf9467d10ab)


- [ ] Also as mentioned earlier, the menu is meant to stick as the user scroll on it. I'll suggest you fix that too. :+1: :+1: 

-  VIDEO REFERENCE:

https://github.com/Danny-imo/my-project/assets/107632780/ebfada4a-1c60-463a-b93e-cf4891b72797